lib/grizzly/zwave/smart_start/meta_extension.ex

```diff
@@ -369,7 +369,8 @@ defmodule Grizzly.ZWave.SmartStart.MetaExtension do
   end
 
   defp do_parse(
-         <<@network_status::size(7), 0::size(1), 0x02, node_id, status, rest::binary>>,
+         <<@network_status::size(7), 0::size(1), 0x04, node_id, status,
+           _long_range_node_id::size(16), rest::binary>>,
```

Comment:
I think need to account for `zipgateway` instances that do not have LR enabled. So, we still need to match the packet does not use the LR encoding.

Reply:
Fixed